File: Java/Iniciante/1004 - Produto Simples..java
/*
* Autor: Pierre Vieira
* Data de submissão: 19/01/2020 16:58:00
*/
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int a, b;
        a = teclado.nextInt();
        b = teclado.nextInt();
        System.out.println("PROD = "+(a*b));
    }
}

File: C#/Iniciante/1004 - Produto Simples.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 11:14:34
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int n1, n2;
            n1 = int.Parse(Console.ReadLine());
            n2 = int.Parse(Console.ReadLine());
            Console.WriteLine("PROD = "+(n1 * n2));
        }
    }
}